addons/mail/static/src/core/common/store_service.js

```diff
@@ -183,6 +183,36 @@ export class Store extends BaseStore {
         return tab === "chat" ? ["chat", "group"] : [tab];
     }
 
+    async onClickHandler(ev, thread, partnerId) {
```

Comment:
I would rename the method to something a little bit more specific, more related to finding dynamic links.

`handleClickOnLink` maybe?